[10-Jan-25 18:36:35 - INFO] - Yumeko - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[10-Jan-25 18:36:35 - INFO] - Yumeko - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[10-Jan-25 18:36:36 - INFO] - Yumeko - Loaded 51 modules: 𝖠𝖽𝗆𝗂𝗇, 𝖠𝖿𝗄, 𝖠𝗇𝗂𝗆𝖾, 𝖠𝗇𝗂𝗆𝖾 𝖲𝖼𝗁𝖾𝖽𝗎𝗅𝖾, 𝖠𝗇𝗇𝗈𝗎𝗇𝖼𝖾𝗆𝖾𝗇𝗍, 𝖠𝗇𝗍𝗂𝖢𝗁𝖺𝗇𝗇𝖾𝗅, 𝖠𝗇𝗍𝗂𝖥𝗅𝗈𝗈𝖽, 𝖠𝗉𝗉𝗋𝗈𝗏𝖺𝗅, 𝖠𝗌𝗄, 𝖡𝖺𝗇, 𝖡𝗅𝖺𝖼𝗄𝗅𝗂𝗌𝗍, 𝖢𝗁𝖺𝗍𝖻𝗈𝗍, 𝖢𝗅𝖾𝖺𝗇𝖾𝗋, 𝖢𝗈𝗌𝗉𝗅𝖺𝗒, 𝖢𝗈𝗎𝗉𝗅𝖾, 𝖣𝗈𝗐𝗇𝗅𝗈𝖺𝖽𝖾𝗋, 𝖤𝗑𝗍𝗋𝖺𝗌, 𝖥𝗂𝗅𝗍𝖾𝗋𝗌, 𝖥𝗈𝗇𝗍, 𝖥𝗎𝗇, 𝖦𝗋𝖾𝖾𝗍𝗂𝗇𝗀𝗌, 𝖨𝖣, 𝖨𝗆𝖺𝗀𝖾, 𝖨𝗆𝗉𝗈𝗌𝗍𝖾𝗋, 𝖩𝗈𝗂𝗇 𝖱𝖾𝗊𝗎𝖾𝗌𝗍, 𝖫𝗈𝖼𝗄𝗌, 𝖫𝗈𝗀 𝖢𝗁𝖺𝗇𝗇𝖾𝗅, 𝖫𝗈𝗀𝗈, 𝖬𝖨𝖢𝖲, 𝖬𝖺𝗌𝗌𝖠𝖼𝗍𝗂𝗈𝗇𝗌, 𝖬𝗎𝗌𝗂𝖼, 𝖭𝖾𝗐𝗌, 𝖭𝗂𝗀𝗁𝗍𝗆𝗈𝖽𝖾, 𝖯𝖺𝗌𝗍𝖾, 𝖯𝗈𝗄𝖾𝖽𝖾𝗑, 𝖰𝗎𝗈𝗍𝖾𝗌, 𝖰𝗎𝗈𝗍𝗅𝗒, 𝖱𝖾𝗏𝖾𝗋𝗌𝖾 𝖲𝖾𝖺𝗋𝖼𝗁, 𝖲𝖾𝗌𝗌𝗂𝗈𝗇, 𝖲𝗉𝗈𝗋𝗍𝗌, 𝖲𝗍𝗂𝖼𝗄𝖾𝗋𝗌, 𝖳𝖳𝖲, 𝖳𝖺𝗀 𝖠𝗅𝗅, 𝖳𝗂𝗇𝗒, 𝖳𝗋𝖺𝗇𝗌𝗅𝖺𝗍𝗂𝗈𝗇, 𝖴𝗉𝗅𝗈𝖺𝖽𝖾𝗋, 𝖴𝗉𝗌𝖼𝖺𝗅𝖾, 𝖴𝗋𝖻𝖺𝗇 𝖣𝗂𝖼𝗍𝗂𝗈𝗇𝖺𝗋𝗒, 𝖶𝖺𝗋𝗇, 𝖶𝗂𝗌𝗁, 𝖶𝗋𝗂𝗍𝖾
[10-Jan-25 18:36:36 - ERROR] - Yumeko - no such column: number
Traceback (most recent call last):
  File "/root/Yumeko/Yumeko/__main__.py", line 265, in <module>
    app.start()
  File "/root/Yumeko/myenv/lib/python3.10/site-packages/pyrogram/sync.py", line 67, in async_to_sync_wrap
    return loop.run_until_complete(coroutine)
  File "uvloop/loop.pyx", line 1518, in uvloop.loop.Loop.run_until_complete
  File "/root/Yumeko/myenv/lib/python3.10/site-packages/pyrogram/methods/utilities/start.py", line 59, in start
    is_authorized = await self.connect()
  File "/root/Yumeko/myenv/lib/python3.10/site-packages/pyrogram/methods/auth/connect.py", line 41, in connect
    await self.load_session()
  File "/root/Yumeko/myenv/lib/python3.10/site-packages/pyrogram/client.py", line 801, in load_session
    await self.storage.open()
  File "/root/Yumeko/myenv/lib/python3.10/site-packages/pyrogram/storage/file_storage.py", line 82, in open
    self.update()
  File "/root/Yumeko/myenv/lib/python3.10/site-packages/pyrogram/storage/file_storage.py", line 51, in update
    version = self.version()
  File "/root/Yumeko/myenv/lib/python3.10/site-packages/pyrogram/storage/sqlite_storage.py", line 297, in version
    return self.conn.execute(
sqlite3.OperationalError: no such column: number
